Jetcaster/wear/src/main/java/com/example/jetcaster/WearApp.kt

```diff
@@ -90,11 +93,10 @@ fun WearApp() {
                 ) {
                     LatestEpisodesScreen(
                         playlistName = stringResource(id = R.string.latest_episodes),
-                        // TODO implement change speed
-                        onChangeSpeedButtonClick = {},
                         onPlayButtonClick = {
                             navController.navigateToPlayer()
-                        }
+                        },
+                        onErrorDialogCancelClick = { navController.popBackStack() }
```

Comment:
These feels a bit too specific, but I'm not sure of MAD guidance here, so happy to go with this.

I guess I'm asking between

- onErrorDialogCancelClick (specific, likely to change)
- onDismiss (user intent, not tied to navigation)
- onNavigateBack (clear the navigation intent)

Reply:
will change to `onDismiss`